build.sbt

```diff
@@ -5,7 +5,7 @@
 name := """cmak"""
 
 /* For packaging purposes, -SNAPSHOT MUST contain a digit */
-version := "3.0.0.5"
+version := "3.0.0.6"
```

Comment:
@pranavbhole @patelh Could you please make github release for the changed app version?